api/app/routers/hfi_calc.py

```diff
@@ -230,9 +231,16 @@ async def download_result_pdf(request: HFIResultRequest,
         # See: https://jinja.palletsprojects.com/en/3.0.x/api/?highlight=functionloader#jinja2.FunctionLoader
         jinja_env = Environment(loader=FunctionLoader(get_template), autoescape=True)
 
-        pdf_bytes = generate_pdf(response, fire_centres_list, jinja_env)
+        username = token.get('preferred_username', None)
```

Comment:
We should add a test for this - I can help. You'd just be checking to see you get a 200.

Reply:
Yeah I don't know how to write those tests